cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/executor/ExecutionTaskManagerTest.java

```diff
@@ -55,18 +56,22 @@ public void testStateChangeSequences() {
     // Cannot rollback.
     testSequences.add(Arrays.asList(IN_PROGRESS, DEAD));
 
+    ReplicaPlacementInfo r0 = new ReplicaPlacementInfo(0);
+    ReplicaPlacementInfo r1 = new ReplicaPlacementInfo(1);
+    ReplicaPlacementInfo r2 = new ReplicaPlacementInfo(2);
```

Comment:
Can we move this changes to a `master` branch patch?

Reply:
`master` branch does not have `ReplicaPlacementInfo` since there is no logdir information for `Replica`.

Reply:
Can we move the broker ids such as `1` to variables as we did here? I.e. let's do the part#2 of: (1) using `ReplicaPlacementInfo`, and (2) moving the hardcoded values to fields in `master`.